qa/erasure-code/ec-rados-plugin=jerasure-k=2-m=2-crush.yaml

```diff
@@ -0,0 +1,52 @@
+roles: 
+- [mon.a, mgr.y, osd.0]
+- [mon.b, osd.1] 
+- [mon.c, osd.2] 
+- [mgr.x, osd.3, client.0]
+overrides:
+  ceph:
+    conf:
+      osd:
+        osd shutdown pgref assert: true
+tasks:
+- rados:
+    clients: [client.0]
+    ops: 400000
+    max_seconds: 600
+    max_in_flight: 64
+    objects: 1024
+    size: 16384
+    ec_pool: true
+    write_append_excl: false
+    erasure_code_crush:
+      name: jerasure22crush
+      id: 22
+      type: erasure
+      min_size: 3
+      max_size: 6
+      steps: [
+        'set_chooseleaf_tries 5',
+        'set_choose_tries 100',
+        'take default class hdd',
+        'choose indep 4 type host',
+        'chooseleaf indep 8 type osd',
```

Comment:
Use choose rather than chooseleaf here due to https://tracker.ceph.com/issues/62213

Reply:
I'm not sure why this one needs 4 hosts, 8 osds per host.  2+2 should probably be a normal host chooseleaf rule, right?

Reply:
isn't it supposed to be 4 host 1 osd? so it will simulate full host shutdown?

Reply:
Ah, 
```
        'choose indep 4 type host',
        'choose indep 1 type osd',
```

is a reasonable interpretation, but you should use ```step chooseleaf indep 4 type host``` instead.  Using two choose steps instead of a single chooseleaf step actually has different behavior once OSDs get marked out.

Reply:
Done